app/src/main/java/com/sdp/movemeet/chat/Message.java

```diff
@@ -8,20 +8,22 @@
     private String messageText;
     private String messageUserId;
     private long messageTime;
+    private String imageUrl;
 
     public Message() {
     }
 
-    public Message(String messageUser, String messageText, String messageUserId) {
+    public Message(String messageUser, String messageText, String messageUserId, String imageUrl) {
 
-        if (messageUser == null || messageText == null || messageUserId == null) {
+        if (messageUser == null || messageUserId == null) {
```

Comment:
Why we don't need to check messageText anymore ?

Reply:
This is because in case we send an image (and not a text), then the messageText field is in fact "null". In short, sending an image is a special case in which, messageText has to be inevitably "null"

Reply:
Okey but does he still have a "safeguard" to avoid sending completely empty messages? I may have looked at it wrong

Reply:
Yes, we still can't send empty message because I check in send sendMessage that the string messageText has to have a length greater than zero. This acts as a safeguard and it's hence not possible to send an empty text message!